jena-fuseki2/jena-fuseki-ui/.eslintrc.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'vue/custom-event-name-casing': 'off'
+    'vue/custom-event-name-casing': 'off',
+    'vue/multi-word-component-names': 'off'
```

Comment:
When we update the linter, and the Vue plug-ins for the eslinter, it's normal to see new rules that were incorporated by the plug-in, but that are just really annoying for us :-) in cases like this I simply disable it, unless it's a rule that appears to be useful.